mkosi/__init__.py

```diff
@@ -3721,20 +3722,63 @@ def make_tar(args: CommandLineArguments, root: str, do_run_build_script: bool, f
     if for_cache:
         return None
 
-    tar_root_dir = f"{root}/usr" if args.usr_only else root
+    root_dir = f"{root}/usr" if args.usr_only else root
```

Comment:
Maybe rename the function make_tar() → make_archive(). The log message already talks about "archive" anyway...

Reply:
I changed tar→archive in various places to make the code more similar in `make_tar` and `make_cpio`. But this function should still be `make_tar`, because it creates a specific type of archive, and there are other functions for other types.